examples/bin/supervise

```diff
@@ -179,13 +195,23 @@ usage() unless GetOptions(
   'vardir|d=s',
   'kill-timeout|t=i',
   'chdir=s',
-  'svlogd:s'
+  'svlogd:s',
+  'command=s@'
 );
 
-usage() unless $opt{'conf'} && $opt{'vardir'};
+usage() unless (($opt{'conf'} || @{$opt{'command'}}) && $opt{'vardir'});
```

Comment:
Ah, this should be:

```
usage() unless (($opt{'command'} && @{$opt{'command'}}) || $opt{'conf'}) && $opt{'vardir'}
```

breaking down `($opt{'command'} && @{$opt{'command'}})`:

- `$opt{'command'}` means "there is some value at `$opt{'command'}`
- we expect this to be an arrayref, so, `@{$opt{'command'}}` means "the arrayref at `$opt{'command'}` is nonempty"

Reply:
Done.